curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc17d3bf86c76b9248c6eb4.md

```diff
@@ -7,9 +7,9 @@ dashedName: step-3
 
 # --description--
 
-Paragraph (`p`) elements are used to create paragraph text on websites.
+The `p` element is used to create a paragraph of text on websites. Create a `p` element below your `h2` element and give it the following text:
```

Comment:
Why were the changes in this file added?

Reply:
Sorry for that I am new to git 
I started working on another issue and didn't realize that the commit was added here . I will take care from next time

Reply:
How do I fix this now should I revert the changes I have made and then commit again ?

Reply:
I see. I would be fine leaving it on this PR if you want, and it can then close both of those issues. Or, if you want more practice, you can remove it. Either way works for me.

Reply:
Okay, I will add the other issue to this PR.